Product/Production/Services/AdminDistributionCore/src/main/java/gov/hhs/fha/nhinc/admindistribution/inbound/AbstractInboundAdminDistribution.java

```diff
@@ -44,7 +44,7 @@
     protected AdminDistributionUtils adminUtils = AdminDistributionUtils.getInstance();
     protected AdapterAdminDistributionProxyObjectFactory adapterFactory = new AdapterAdminDistributionProxyObjectFactory();
 
-    abstract void processAdminDistribution(EDXLDistribution body, AssertionType assertion);
+    public abstract void processAdminDistribution(EDXLDistribution body, AssertionType assertion);
```

Comment:
Within this PR, there are quite a few functions that have been changed from package-private to public; just curious, what's the reasoning behind those changes?